mmv1/third_party/terraform/resources/resource_sql_database_instance.go.erb

```diff
@@ -1055,27 +1055,25 @@ func resourceSqlDatabaseInstanceCreate(d *schema.ResourceData, meta interface{})
 	// Users in a replica instance are inherited from the master instance and should be left alone.
 	// This deletion is done immediately after the instance is created, in order to minimize the
 	// risk of it being left on the instance, which would present a security concern.
-	if sqlDatabaseIsMaster(d) {
-		var users *sqladmin.UsersListResponse
+	if sqlDatabaseIsMaster(d) && strings.Contains(strings.ToUpper(databaseVersion), "MYSQL") {
```

Comment:
Is `MYSQL` the only database with this behavior? Assuming so because of your change

Reply:
Yes, Only MySQL has the option of password-less `root@%` user as the default root user. Postgresql has a default user called `postgres` and Sql Server has a default user called `sqlserver`, and both requires a password.